content/courses/program-security/closing-accounts.md

```diff
@@ -165,73 +145,6 @@ pub struct Data {
 }
 ```
 
-Note that the example above is using Anchor's `CLOSED_ACCOUNT_DISCRIMINATOR`.
```

Comment:
Likewise tell me about why you got rid of this section.

Reply:
“CLOSED_ACCOUNT_DISCRIMINATOR” was removed in the latest version of anchor-lang
https://github.com/coral-xyz/anchor/pull/2726#issue-2038294408

Reply:
Brilliant! You can resolve this (also if you see any other references to `CLOSED_ACCOUNT_DISCRIMINATOR`)